src/main/functions/DidJWT.ts

```diff
@@ -74,10 +73,7 @@ export async function createDidJWT(
   return createJWT(payload, { issuer, signer, alg: header.alg, expiresIn, canonicalize }, header);
 }
 
-export async function signDidJwtPayload(
-  payload: AuthenticationRequestPayload | AuthenticationResponsePayload,
-  opts: AuthenticationRequestOpts | AuthenticationResponseOpts
-) {
+export async function signDidJwtPayload(payload: IdTokenPayload, opts: AuthenticationRequestOpts | AuthenticationResponseOpts) {
```

Comment:
I don't get why something that was signing 2 JWT strings as it should be now is signing what seems to be a JSON object

Reply:
Fixed!